requirements.txt

```diff
@@ -1,10 +1,14 @@
-fastapi==0.68.1
+aiobotocore==1.4.2
+aiofiles==0.7.0
+fastapi==0.70.0
 imjoy-jupyter-extension==0.2.17
 imjoy-rpc==0.3.30
 ipykernel==5.5.5
+jinja2==3.0.1
 jupyter==1.0.0
 msgpack==1.0.2
 numpy==1.19.5  # needs to stay compatible with latest tensorflow
+playwright==1.15.0
```

Comment:
playwright requires Python 3.7+.

Reply:
I think we can make 3.7 as minimal requirement now.